t/app/controller/waste_kingston.t

```diff
@@ -835,9 +827,32 @@ FixMyStreet::override_config {
         $mech->content_like(qr#/waste/12345">Show upcoming#, "contains link to bin page");
     };
 
+    subtest 'sacks, cannot modify, but can buy more' => sub {
+        set_fixed_time('2021-01-09T17:00:00Z');
+        $mech->log_in_ok($user->email);
+        $mech->get_ok('/waste/12345');
+        $mech->content_lacks('Modify your garden waste subscription');
+        $mech->content_contains('Order more garden sacks');
+        $mech->get_ok('/waste/12345/garden_modify');
+        $mech->content_contains('<span id="pro_rata_cost">41.00');
+        $mech->submit_form_ok({ with_fields => { name => 'Test McTest' } });
+        $mech->submit_form_ok({ with_fields => { tandc => 1 } });
```

Comment:
Should this maybe check to make sure there isn't a field to alter the number of sacks, just to be on the safe side.